### PR TITLE
Fixed wrong API end point reference

### PR DESCRIPTION
**Pull request by obermaierc**
**- Summary**

I came across this description how to call the endDelay endpoint while browsing the docs.  
Guess it should be "automation" instead of "authentication".

**- Test plan**

The URL http://zap/UI/automation/action/endDelayJob/ exists, while http://zap/UI/authentication/action/endDelayJob/ does not.

**- Description for the changelog**

Exchanged the string "authentication" for "automation". That's it.

### CODE REVIEW

**Comment by thc202** [comment]
The commit should be signed off: https://github.com/zaproxy/zaproxy-website/pull/1059/checks?check_run_id=6857685840

**Comment by thc202** [comment]
btw, this should really be fixed in the source not here (as mentioned in the file `This page was generated from the add-on.`), but I guess it's fine to go ahead with the change anyway.

**Review by kingthorin — APPROVED** [review]

**Comment by obermaierc** [comment]
Ok, well. I should have seen that notice. Thank you 😄 
I created a new signed-off PR in https://github.com/zaproxy/zap-extensions/pull/3850

**Comment by thc202** [comment]
We can still merge this, no need to wait until the release of the add-on. Since you signed off the other commit we can assume you would here as well, right?

**Comment by obermaierc** [comment]
I screwed that up by deleting my forked repo, so in my understanding this pr has become immutable, right?

Anyway I've forked it again and created a new signed-off commit https://github.com/obermaierc/zaproxy-website/commit/52a8d2645b6f41875e435863ba0853b20718dddc

to merge it, I would have to create another pull request, right?

But I would leave it up to you if I should really do that...

Thank you for being so patient

**Comment by thc202** [comment]
That's ok, you agree with DCO, right? We can set the check to pass and merge as is.

**Comment by obermaierc** [comment]
> That's ok, you agree with DCO, right? We can set the check to pass and merge as is.

Yeah, of course.

That's easier 😃

**Review by thc202 — APPROVED** [review]

**Pull request merged**
